vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -34,12 +42,12 @@ def textual_ipaddress
     {:label => "IP Address", :value => @ems.ipaddress}
   end
 
-  def textual_type
-    {:label => "Type", :value => @ems.emstype_description}
+  def textual_port
+    @ems.supports_port? ? {:label => "Port", :value => @ems.port} : nil
   end
 
-  def textual_port
-    @ems.supports_port? ? {:label => "API Port", :value => @ems.port} : nil
+  def textual_zone_id
+    {:label => "Zone", :value => @ems.zone_id}
```

Comment:
We show a zone name in Smart Management box on Infra Provider summary without a link, we should definitely show a zone name instead of an id here:
    {:label => "Managed by Zone", :image => "zone", :value => @ems.zone.name}